neo-codap/src/case-table/case-table.tsx

```diff
@@ -201,10 +231,36 @@ export class CaseTable extends React.Component<ICaseTableProps, ICaseTableState>
     this.gridColumnDefs = dataSet ? this.getColumnDefs(dataSet) : [];
     this.gridRowData = dataSet ? this.getRowData(dataSet) : [];
     if (this.gridApi) {
+      this.gridRowData.push({id: LOCAL_CASE_ID});
       this.gridApi.setRowData(this.gridRowData);
+      setTimeout(() => this.ensureFocus(dataSet));
+    }
+  }
+
+  ensureFocus = (dataSet?: IDataSet) => {
+    const currentCell = this.gridApi.getFocusedCell();
+    const lastRowIndex = this.gridApi.paginationGetRowCount() - 1;
+    if (!currentCell && (lastRowIndex >= 0) && dataSet && (dataSet.attributes.length > 0)) {
+      const firstColId = dataSet.attributes[0].id;
+      this.gridApi.setFocusedCell(lastRowIndex, firstColId);
+    }
+  }
+
+  focusOnNextRow = () => {
+    const currentCell = this.gridApi.getFocusedCell();
+    if (currentCell) {
+      this.gridApi.setFocusedCell(currentCell.rowIndex + 1, currentCell.column.getColId());
     }
   }
 
+  // tslint:disable-next-line:no-any
```

Comment:
Should we consider disabling this globally? It seems reasonable to me that this could be developer discretion.